src/App.js

```diff
@@ -1,19 +1,23 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Hi @codehass  please remove the eslint-disable-next-line and fix the eslint error